### PR TITLE
IBX-4981 Page Builder for B2B Portals

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-4981
| Versions      | 4.5
| Edition       | Experience, Commerce

Refers to https://github.com/ezsystems/developer-documentation/pull/1971

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by ViniTou — APPROVED** [review]

File: docs/customer_management/build_customer_portal.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+description: Use Page Builder to create and edit Customer Portal.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:2]:
```suggestion
description: Use Page Builder to create and edit a Customer Portal.
```

File: docs/customer_management/build_customer_portal.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+edition: experience
+---
+
+# Create and edit Customer Portal
+
+To create and edit Customer Portal with Page Builder, you need to set it up in configuration.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:8]:
```suggestion
To create and edit a Customer Portal with Page Builder, you need to first set it up in configuration.
```

File: docs/customer_management/build_customer_portal.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+For detailed instructions on how to do it, go to
+[Create Customer Portal]([[= developer_doc =]]/customer_management/cp_page_builder/) in Developer Documentation.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:10]:
```suggestion
[Create Customer Portal]([[= developer_doc =]]/customer_management/cp_page_builder/).
```

File: docs/customer_management/build_customer_portal.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+The Customer Portal creation and edition are based on Page Builder feature and work on the same principles.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:12]:
```suggestion
The Customer Portal creation and edition are based on Page Builder and work on the same principles.
```

File: docs/customer_management/build_customer_portal.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+If you are unfamiliar with how Page Builder works follow [Create and edit Pages](create_edit_pages.md).
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:13]:
```suggestion
If you are unfamiliar with how Page Builder works, see [Create and edit Pages](create_edit_pages.md).
```

File: docs/customer_management/build_customer_portal.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+## Create Customer Portal
+
+To create a new Customer Portal, go to **Content** and from the list select **Content structure**.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:17]:
```suggestion
To create a new Customer Portal, go to **Content** and from the menu select **Content structure**.
```

File: docs/customer_management/build_customer_portal.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+There, you should navigate to the root folder for your Customer Portals.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:18]:
```suggestion
There, navigate to the root folder for your Customer Portals. 
```

File: docs/customer_management/build_customer_portal.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+Remember to specify its `location_id` in the configuration, you will find it under **Details**.
+For more information, see [Configure Page Builder access to Customer Portal.]([[= developer_doc =]]/customer_management/cp_page_builder/#configure-page-builder-access-to-customer-portal)
+
+Inside a root folder you can select **Create content** from the right-side toolbar.
+On the list of Content items, you will see two possibilities **Customer Portal** and **Customer Portal Page**.
+
+![Create content tab](img/cp_portal_vs_page.png)
+
+The first one is a container for your Customer Portal pages (this is not a root folder), and the second one represents the actual page.
+We highly recommend that you use Customer Portal containers to divide and store your portal pages.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:29]:
```suggestion
It is recommended to use Customer Portal containers to divide and store your portal pages.
```

File: docs/customer_management/build_customer_portal.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+
+First, select **Customer Portal** and name it appropriately.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:31]: change to a numbered list?

File: docs/customer_management/build_customer_portal.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+Next, navigate to a newly added container and create **Customer Portal Page**.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:32]:
```suggestion
Next, navigate to the newly added container and create **Customer Portal Page**. 
```

File: docs/customer_management/build_customer_portal.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+
+The first one is a container for your Customer Portal pages (this is not a root folder), and the second one represents the actual page.
+We highly recommend that you use Customer Portal containers to divide and store your portal pages.
+
+First, select **Customer Portal** and name it appropriately.
+Next, navigate to a newly added container and create **Customer Portal Page**. 
+
+![Customer Portal container](img/cp_folder_for_portals.png)
+
+On the **Page creation** modal, you should see the Customer Portal layout where you can
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:36]:
```suggestion
In the **Page creation** box, you should see the Customer Portal layout where you can 
```
"modal" is a technical term, so maybe box/pop-up/screen?

File: docs/customer_management/build_customer_portal.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+On the **Page creation** modal, you should see the Customer Portal layout where you can 
+add dedicated Customer Portal block, Sales Representative, or choose from selection of blocks available to your [[= product_name =]] version.
+
+For a list of blocks available out of the box, see [Block reference](block_reference.md).
+
+![Page Builder view](img/cp_page_builder.png)
+
+If provided ready-to-use Page blocks are not sufficient, you can [add your own blocks]([[= developer_doc =]]/content_management/pages/create_custom_page_block/).
+
+Before you publish or save the Customer Portal page, edit its title and description in the Field view, you can find it in the top-left-side toolbar.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:45]: "top-left-side toolbar" is confusing, even with the accompanying screen

File: docs/customer_management/build_customer_portal.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+If you are ready to publish the Customer Portal page, click **Publish** in the top-right corner.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:47]:
```suggestion
If you are ready to publish the Customer Portal page, click **Publish** in the top right corner.
```

File: docs/customer_management/build_customer_portal.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+You can also save it as a draft, even if some required fields are not filled in, click **Save draft**.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:48]:
```suggestion
You can also save it as a draft, even if some required fields are not filled in.
To do it, click **Save draft**.
```

File: docs/customer_management/build_customer_portal.md [hunk 15]
```diff
@@ -0,0 +1,79 @@
...
+
+## Add multiple pages
+
+You can have multiple Customer Portal pages available in one Customer Portal by adding them under one Customer Portal container.
+If company members have sufficient `content/read` policies and have the portal assigned to their Customer Group, they will see the changes in the left-side menu.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:53]:
```suggestion
If company members have sufficient `content/read` Policies and have the portal assigned to their Customer Group, they will see the changes in the left menu.
```

File: docs/customer_management/build_customer_portal.md [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+
+![Multiple pages in one portal](img/cp_multiple_pages.png)
+
+You can manipulate the order of pages in a menu by assigning priority to them in the **Customer Portal** container under **View**->**Sub-items**.
+
+![Assigning page priority](img/cp_page_priority.png)
+
+## Grant permissions
+
+Company members need to have below permissions to be able to see custom Customer Portals:
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:65]:
```suggestion
Company members need to have the following permissions to be able to see custom Customer Portals:
```

File: docs/customer_management/build_customer_portal.md [hunk 17]
```diff
@@ -0,0 +1,79 @@
...
+
+- `user/login` to `custom_portal` SiteAccess
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:67]: Is it really `custom_portal`, not `customer`?

File: docs/customer_management/build_customer_portal.md [hunk 18]
```diff
@@ -0,0 +1,79 @@
...
+- `content/read` to selected Customer Portals
+
+![Customer Portal permissions](img/cp_permissions.png)
+
+If members of the company don't have sufficient permissions for any Customer Portal, they will be transferred to the default Customer Portal view.
+
+!!! note
+
+    Customer Portal is only available to users that are members of the company. Even if user has all the sufficient permissions but is not a meber of a company, they will not see the Customer Portal.
```
Comment (DominikaK) [inline docs/customer_management/build_customer_portal.md:76]:
```suggestion
    Customer Portal is only available to users that are members of the company.
    Even if a user has all sufficient permissions but is not a member of a company, they will not see the Customer Portal.
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/customer_management/build_customer_portal.md (hunk 12 above)
Comment (MagdalenaZuba) [reply docs/customer_management/build_customer_portal.md:45]: The placement of this toolbar is confusing, and the icons are not helping. The error message just says 'Fields error' and adds a soft red highlight on the Fields button.

**Review by DominikaK — APPROVED** [review]

**Pull request merged**